Parsers/ASimRegistryEvent/test/TrendMicro_VisionOne_vimRegistryEvent_SchemaTest.csv

```diff
@@ -0,0 +1,26 @@
+﻿Result
+"(1) Warning: Missing recommended field [ActorUserId]"
+"(1) Warning: Missing recommended field [DvcIpAddr]"
```

Comment:
no IP Details coming?

Reply:
Did you validate with official documentation? If we are getting values, lets not skip them without validating.

Reply:
In official documentation, it is mentioned that endpoint Ip list will be coming. No oher information is available.
![image](https://github.com/Azure/Azure-Sentinel/assets/107753570/5415b2c1-2a3b-4601-9846-65e8734106b6)